symphony-bdk-legacy/symphony-api-client-java/src/main/java/services/ActionFirehose.java

```diff
@@ -0,0 +1,140 @@
+package services;
+
+import clients.SymBotClient;
+import clients.symphony.api.FirehoseClient;
+import listeners.FirehoseListener;
+import model.DatafeedEvent;
+import model.events.MessageSent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+public class ActionFirehose implements IActionFirehose {
```

Comment:
Our convention in this repo would be to call the interface ActionFirehose and the implementation class ActionFirehoseImpl